Generate/src/Medication-9-0-7/_components/assert-returnEntryCount.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-model href="../../../general/schematron/NictizTestScript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <nts:component  xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript">
+    <nts:parameter name="breakdown" value=""/>
     <nts:parameter name="warningOnly" value="false"/>
     <action>
         <assert>
-            <description value="{$description}"/>
+            <description value="Confirm that the returned searchset Bundle contains {$count} entry(-ies){$breakdown}."/>
```

Comment:
Erop vertrouwen dat het attribute begint met een spatie? Jij bent niet bang :D

Reply:
Ehm... dit was meer om ervoor te zorgen dat bij gebrek aan een waarde er geen overbodige spatie komt te staan. Ik mis een stukje logica "if gevuld, dan spatie" in NTS. Maar je hebt wel een punt. Ik kijk even of ik dit kan bijwerken naar bijv. een losstaande zin.

Reply:
Ohja, daar had ik niet over nagedacht :(